loadflow/loadflow-api/src/main/java/com/powsybl/loadflow/json/LoadFlowParametersDeserializer.java

```diff
@@ -71,16 +69,18 @@ public LoadFlowParameters deserialize(JsonParser parser, DeserializationContext
 
                 case "extensions":
                     parser.nextToken();
-                    extensions = JsonUtil.readExtensions(parser, deserializationContext, JsonLoadFlowParameters.getExtensionSerializers());
+                    if (parameters.getExtensions().size() > 0) {
```

Comment:
As we are in an update operation here (since we have an initial value of parameters, I think it's simpler to call `updateExtensions` in any case. If there is no extension, it will have the same behaviour anyway.